### PR TITLE
feat(JSTempalte): Support JS templates [EXT-3441]

### PR DESCRIPTION
**Pull request by kdamball**
Support template flag for typescript(`--typescript`/`-ts`)  & JS(`--javascript`/`-js`)

Usage: `@contentful/create-contentful-app init my-contentful-app --javascript`

### CODE REVIEW

File: packages/contentful--create-contentful-app/lib/index.js [hunk 1]
```diff
@@ -67,7 +68,16 @@ async function initProject() {
 
     console.log(`Creating a Contentful app in ${chalk.bold(tildify(fullAppFolder))}.`);
 
-    await cloneTemplate('typescript', fullAppFolder);
+    program
+      .option('--javascript, -js')
+      .option('--typescript, -ts')
+      .action(async (opts) => {
+        console.log()
```
Comment (andipaetzold) [inline packages/contentful--create-contentful-app/lib/index.js:75]: 🤔  why is that here?

**Review by andipaetzold — APPROVED** [review]
Seems to work fine 👍

**Review by andipaetzold — APPROVED** [review]

**Comment by andipaetzold** [comment]
I think you need to run `npm i` on root level and then also tests should be green.

**Comment by kdamball** [comment]
Lerna strikes again
